how-to/how-to-set-prometheus-in-k8s.md

```diff
@@ -0,0 +1,59 @@
+# How to monitor Kata Containers in K8s clusters
+
+
+This document will describe how to run `kata-magent` in a K8s cluster, and using Prometheus's K8s service discovery to scrape metrics from `kata-agent`.
+
+
+- [Introduction](#introduction)
+- [Pre-requisites](#pre-requisites)
+- [Configure Prometheus](#configure-prometheus)
+- [Configure `kata-magent`](#configure-kata-magent)
+
+> **Warning**: This how-to is only for evaluation purpose, you **SHOULD NOT** running it in production using this configurations.
+
+## Introduction
+
+If you are running Kata containers in a K8s cluster, the best way to run `kata-magent` is using K8s native `DaemonSet`, `kata-magent` will run on desired K8s nodes without other operations when new nodes joined the cluster.
+
+Prometheus also support a K8s service discovery that can find scrape targets dynamically without explicitly setting `kata-magent`'s metric endpoints.
+
+## Pre-requisites
+
+You must have a running K8s cluster first. If not, [install a K8s cluster](https://kubernetes.io/docs/setup/) first.
+
+Also you should ensure that `kubectl` working correctly.
+
+## Configure Prometheus
+
+Running Prometheus is easy.
```

Comment:
I agree its easy, but let's not qualify the ease of use, as for some people its trivial, and for other people it'll be the hardest thing they ever do. :D

```suggestion
Start Prometheus by utilizing our sample manifest:  
```